frontend/components/App/App.tsx

```diff
@@ -111,6 +112,12 @@ const App = ({ children, location }: IAppProps): JSX.Element => {
           } catch (error) {
             console.error(error);
           }
+          try {
+            const vppInfo = await mdmAppleAPI.getVppInfo();
+            setVppExpiry(vppInfo.renew_date);
+          } catch (e) {
+            console.log(e);
```

Comment:
Did you mean to keep the console log here?

Reply:
yes, for now its following the pattern of the request before it. I am working on another PR to move around all this so I'll be removing it soon.